lib/cherry_silt/armor.rb

```diff
@@ -16,10 +16,13 @@
 
 require 'cherry_silt/item'
```

Comment:
other files don't seem to require this line. for some reason armor fails if it's not here. any ideas?